shell/platform/darwin/common/framework/Source/FlutterStandardCodec.mm

```diff
@@ -221,115 +230,142 @@ - (void)dealloc {
   [super dealloc];
 }
 
-- (void)writeByte:(UInt8)value {
-  [_data appendBytes:&value length:1];
+static void WriteByte(CFMutableDataRef data, UInt8 value) {
```

Comment:
Should this have a FLT prefix?

Reply:
It's not required because these are static functions, their visibility is limited to this compilation unit so they can't collide.